Sprint-2/errors/0.js

```diff
@@ -4,6 +4,8 @@
 // interpret the error message and figure out why an error is occurring
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+  let str1 = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
```

Comment:
What will be returned here?

Write down your prediction first.

Reply:
with this current codes it will return the same string as the input, which is not the thing was needed!
so I amended the last line . It must be "return str1"